DemoApp/CallView.swift

```diff
@@ -34,6 +33,24 @@ struct CallView: View {
                     }
                 }
             )
+            .onAppear { joinCall(with: callId) }
```

Comment:
This can be confusing for our integrators, since it looks like we always join a call on appear. Maybe we can update the method name to `joinCallIfNeeded`?